code_challenges/code_wars/two_sum.py

```diff
@@ -0,0 +1,5 @@
+def two_sum(numbers, target):
+    for i in range(len(numbers)):
+        for j in range(i + 1, len(numbers)):
+            if numbers[i] + numbers[j] == target:
+                return [i, j]
```

Comment:
This is fine, it's an O(n^2) solution.  Consider this:
```suggestion
    seen = {}
    for i, num in enumerate(numbers):
        if target - num in seen:
            return [i, seen[target - num]]
        seen[num] = i
```